### ISSUE TITLE
Improve VoiceOver interaction with Strike Zone Map

### ISSUE THREAD

**Issue by kellylford** [kellylford/Scores#42]
The Strike Zone Map feature in the MLB Plays tab does not work well with VoiceOver. The UI entry point has been removed for now. This issue tracks improving the VoiceOver experience so the feature can be re-enabled.
